src/FileCatalog/ComplexNumbers/Program.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+
+namespace ComplexNumbers
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Program program = new Program();
+
+            Console.WriteLine("Enter a");
+            double a = program.ChangedLine();
+            Console.WriteLine("Enter b");
+            string b = Convert.ToString(program.ChangedLine()) + "i";
+            ComplexNumbers complexNumber = new ComplexNumbers(a, b);
+            complexNumber.NumToString();
+            const string addControl = "A";
+            const string magnitudeControl = "M";
+            const string stringControl = "S";
+            const string exitControl = "E";
+
+            bool isExit = false;
+            string input;
+
+            while (!isExit)
+            {
+                input = Console.ReadLine();
+                switch (input)
+                {
+                    case magnitudeControl:
+                        {
```

Comment:
you have to fix formating:
                    case magnitudeControl:
                        {
like this
                    case magnitudeControl:
                    {

please, see
https://msdn.microsoft.com/ru-ru/library/06tc147t.aspx